modules/nf-core/filtlong/tests/main.nf.test

```diff
@@ -0,0 +1,93 @@
+nextflow_process {
+
+    name "Test Process FILTLONG"
+    script "../main.nf"
+    process "FILTLONG"
+    config "./nextflow.config"
```

Comment:
Oh, that's what I needed for converting the `bcl2fastq` tests! :grinning:

Reply:
You can also specify on the test-level, so if you need different settings, just provide multiple nextflow.config files